### ISSUE TITLE
Use Router Link

### ISSUE THREAD

**Issue by Vict0rSch** [Vict0rSch/metada#5]
switch from button + onClick to `<Link to="...">...</Link>`

**Comment by kamry-bowman** [comment]
Could I take a swing at this one?

**Comment by Vict0rSch** [comment]
Please do :) 

My understanding is that using more <Link> is also better for SEO ; any opinion?

**Comment by kamry-bowman** [comment]
Yes, I think in general you can't rely on web crawlers to interact with your javascript (though people disagree), but Link tags are built on top of <a> tags, and web crawlers are pretty guaranteed to follow those. That's my understanding anyway!

**Comment by Vict0rSch** [comment]
Yeah makes sense I agree.

Thanks a lot for your effort on <Link>s 👍 

Would you mind doing so on the `dev` branch? That's where I develop the project right now :)
